service/storage_service.cc

```diff
@@ -1352,6 +1352,21 @@ std::list<gms::inet_address> storage_service::get_ignore_dead_nodes_for_replace(
     return ignore_nodes;
 }
 
+future<std::unordered_set<gms::inet_address>> storage_service::get_nodes_to_sync_with(
```

Comment:
See Benny's patch here https://github.com/scylladb/scylladb/pull/13256. He has a refactor in this area.

Reply:
which now badly conflicts with this change... :-/